lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -380,20 +381,29 @@ private IConfiguration GetRuntimeConfiguration(string runOnYarn, int numberOfEva
             {
                 case Local:
                     var dir = Path.Combine(".", runtimeFolder);
-                    return LocalRuntimeClientConfiguration.ConfigurationModule
+                    var localClientConfig = LocalRuntimeClientConfiguration.ConfigurationModule
                         .Set(LocalRuntimeClientConfiguration.NumberOfEvaluators, numberOfEvaluator.ToString())
                         .Set(LocalRuntimeClientConfiguration.RuntimeFolder, dir)
                         .Build();
+                    return Configurations.Merge(localClientConfig, GetTcpConfiguration());
                 case YARN:
                     var yarnClientConfig = YARNClientConfiguration.ConfigurationModule.Build();
                     var tcpPortConfig = TcpPortConfigurationModule.ConfigurationModule
                        .Set(TcpPortConfigurationModule.PortRangeStart, PortRangeStart)
                        .Set(TcpPortConfigurationModule.PortRangeCount, PortRangeCount)
                        .Build();
-                    return Configurations.Merge(yarnClientConfig, tcpPortConfig);
+                    return Configurations.Merge(yarnClientConfig, tcpPortConfig, GetTcpConfiguration());
                 default:
                     throw new Exception("Unknown runtime: " + runOnYarn);
             }
         }
+
+        private IConfiguration GetTcpConfiguration()
```

Comment:
I'd call this method `GetTcpConnectionConfiguration` (it's used next to `tcpPortConfig` and can get a bit confusing)